packages/babylon/test/fixtures/core/uncategorised/558/actual.js

```diff
@@ -0,0 +1 @@
+const let = 4;
```

Comment:
This isn't valid (https://tc39.github.io/ecma262/#sec-let-and-const-declarations-static-semantics-early-errors)

Reply:
cool, I'll just get rid of this test